src/quo/components/loaders/skeleton_list/constants.cljs

```diff
@@ -33,30 +35,30 @@
                       :message {:width         156
                                 :height        16
                                 :margin-bottom 0}}}
-   :list-items    {0 {:author  {:width         112
-                                :height        8
-                                :margin-bottom 0}
-                      :message {:width         144
+   :list-items    {0 {:author  {:width         144
                                 :height        16
-                                :margin-bottom 8}}
-                   1 {:author  {:width         96
+                                :margin-bottom 8}
+                      :message {:width         112
                                 :height        8
-                                :margin-bottom 0}
-                      :message {:width         212
+                                :margin-bottom 0}}
+                   1 {:author  {:width         212
                                 :height        16
-                                :margin-bottom 8}}
-                   2 {:author  {:width         80
+                                :margin-bottom 8}
+                      :message {:width         96
                                 :height        8
-                                :margin-bottom 0}
-                      :message {:width         249
+                                :margin-bottom 0}}
+                   2 {:author  {:width         249
                                 :height        16
-                                :margin-bottom 8}}
-                   3 {:author  {:width         124
+                                :margin-bottom 8}
+                      :message {:width         80
                                 :height        8
-                                :margin-bottom 0}
-                      :message {:width         156
+                                :margin-bottom 0}}
+                   3 {:author  {:width         156
                                 :height        16
-                                :margin-bottom 8}}}
+                                :margin-bottom 8}
+                      :message {:width         124
+                                :height        8
+                                :margin-bottom 0}}}
```

Comment:
all of these `def`s could be inside a vector, instead of a integer indexed hash-map

A vector is easier to maintain, if we add a key we don't need to set/change the indexing, and can also be accessed using `get`.

do those indexes (1,2,3...) mean something to us? like status-go IDs or something?

Reply:
> A vector is easier to maintain, if we add a key we don't need to set/change the indexing, and can also be accessed using get.

True! I believe the `skeleton-list` uses hash maps as it performs better compared to the vector in `get`. This was mainly due to improving the performance of opening animations of the chat (uses `skeleton-list` with `messages` type) from the `jump-to` screen.

> do those indexes (1,2,3...) mean something to us? like status-go IDs or something?

The indexes (0,1,2,3) are correlated with `pattern-type` in the design.

Reply:
OK thanks for the explanation! :)